src/components/mortgage-investment-compare/amortization-table.js

```diff
@@ -184,6 +204,14 @@ const AmortizationTable = props => {
           })}
         </Tbody>
       </Table>
+      {props.includeInflation && (
+        <small>
+          <b>
+            ** If you include inflation, your payments will never actually
+            become smaller; they will have a smaller purchasing power.
+          </b>
+        </small>
```

Comment:
small disclaimer I added at the bottom of the table. Chris told me that it would be confusing to some people to see their payments decrease, because that's not what's actually happening, so just clarifying. Only appears when you include inflation.